src/components/Data/Documents/Page.vue

```diff
@@ -390,6 +405,41 @@ export default {
     }
   },
   methods: {
+    async subscribeToCurrentDocs() {
+      try {
+        if (this.subscribeRoomId) {
+          await this.$kuzzle.realtime.unsubscribe(this.subscribeRoomId)
+          this.subscribeRoomId = null
+        }
+        const roomId = await this.$kuzzle.realtime.subscribe(
+          this.indexName,
+          this.collectionName,
+          {
+            or: this.documents.map(d => ({ equals: { _id: d._id } }))
```

Comment:
Oh... This means we don't get to see newly created documents?

Reply:
The goal of that feature is only to see the current listed documents being updated
I didn't want to update the list while an use is browsing it